pkg/snapshotter/snapshotter_csi.go

```diff
@@ -1052,3 +1083,174 @@ func (c *csiDriver) CleanUpRestoredResources(backupLocation *storkapi.BackupLoca
+			{
+				APIGroups:     []string{"security.openshift.io"},
+				Resources:     []string{"securitycontextconstraints"},
+				ResourceNames: []string{"hostaccess"},
+				Verbs:         []string{"use"},
+			},
+		},
+	}
+}
+
+func (c *csiDriver) deletejob(pvc *v1.PersistentVolumeClaim, namespace string) error {
+	jobName := toBoundJobPVCName(pvc.Name, string(pvc.GetUID()))
+	t := func() (interface{}, bool, error) {
+		if err := batch.Instance().DeleteJobWithForce(jobName, namespace); err != nil && !k8s_errors.IsNotFound(err) {
+			return nil, true, fmt.Errorf("deletion of job %s/%s failed: %v", namespace, jobName, err)
+		}
+		if err := utils.CleanServiceAccount(jobName, namespace); err != nil {
+			return nil, true, fmt.Errorf("deletion of service account %s/%s failed: %v", namespace, jobName, err)
+		}
+
+		if _, err := batch.Instance().GetJob(jobName, namespace); err != nil && !k8s_errors.IsNotFound(err) {
+			if err == nil {
+				errMsg := fmt.Sprintf("job %s/%s still exists after deletion: %v", namespace, jobName, err)
+				return nil, true, fmt.Errorf(errMsg)
+			}
+		}
+
+		if _, err := core.Instance().GetServiceAccount(jobName, namespace); err != nil && !k8s_errors.IsNotFound(err) {
+			if err == nil {
+				errMsg := fmt.Sprintf("service account %s/%s still exists after deletion: %v", namespace, jobName, err)
+				return nil, true, fmt.Errorf(errMsg)
+			}
+		}
+
+		pods, err := core.Instance().GetPodsUsingPVC(pvc.Name, namespace)
+		if err == nil && len(pods) > 0 {
+			logrus.Debugf("pvc %s/%s is still getting used by job pod %s", namespace, pvc.Name, pods[0].Name)
+			return nil, true, fmt.Errorf("pvc %s/%s is still getting used by job pod %s", namespace, pvc.Name, pods[0].Name)
+		}
+
+		return nil, false, nil
+	}
+	if _, err := task.DoRetryWithTimeout(t, shortRetryTimeout, shortRetryTimeoutInterval); err != nil {
+		return fmt.Errorf("timed out waiting for cleaning up job %s related resources used for volume bind for pvc: %v", jobName, err)
+	}
+	return nil
+}
+
+func buildJobSpec(
+	pvc *v1.PersistentVolumeClaim,
+	namespace string,
+) (*batchv1.Job, error) {
+	jobName := toBoundJobPVCName(pvc.Name, string(pvc.GetUID()))
+	cmd := strings.Join([]string{
+		"sleep 5",
+	}, " ")
+
+	jobPodBackOffLimit := int32(1)
+
+	job := &batchv1.Job{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      jobName,
+			Namespace: namespace,
+			Annotations: map[string]string{
+				utils.SkipResourceAnnotation: "true",
+			},
+		},
+		Spec: batchv1.JobSpec{
+			BackoffLimit: &jobPodBackOffLimit,
+			Template: v1.PodTemplateSpec{
+				Spec: v1.PodSpec{
+					RestartPolicy:      v1.RestartPolicyOnFailure,
+					ImagePullSecrets:   utils.ToImagePullSecret(utils.KopiaExecutorImageSecret(kdmpConfig, "kube-system")),
+					ServiceAccountName: jobName,
+					Containers: []v1.Container{
+						{
+							Name:            "kopiaexecutor",
+							Image:           utils.KopiaExecutorImage(kdmpConfig, "kube-system"),
+							ImagePullPolicy: v1.PullAlways,
```

Comment:
Make this as `PullIfNotPresent` and also `KdmpConfigmapNamespace` instead of "kube-system"

Reply:
done